dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hadoop/HadoopMetadataConnector.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.google.edwmigration.dumper.application.dumper.connector.cloudera;
+package com.google.edwmigration.dumper.application.dumper.connector.hadoop;
```

Comment:
In my understanding Hadoop is generic ecosystem and Cloudera, Mapr, Hortonworks are Hadoop vendors, kind of subset of Hadoop.
So, I will expect the following packages structure:
```
hadoop/
generic hadoop classes/connectors
  cloudera/
  cloudera specific
    manager/
      clouder manager (existing connectors)
  mapr/
```

Reply:
That could make sense.

I will, however, leave moving those folders to a different PR later on. To avoid mix-ups and make it easy to rollback this PR if necessary.